pkg/model/ordered_dict.go

```diff
@@ -91,6 +92,23 @@ func CleanTypes(inter interface{}) interface{} {
 		}
 
 		return tab
+
+	case json.Number:
+
+		resFloat64, err := typedInter.Float64()
+		if err == nil {
+			return resFloat64
+		}
+
+		return typedInter.String()
+
+	case uint64:
+		res := float64(typedInter)
+		return res
+
+	case int64:
+		res := float64(typedInter)
+		return res
```

Comment:
What about other types ? (int, int8, int16, in32, uint, uint8, uint16, uint32, uint64, rune, byte, ...)

Reply:
We expect to never see anything else than uint64 and int64 maybe ?

Reply:
yes